projects/microphysics/train/argo.yaml

```diff
@@ -27,6 +30,8 @@ spec:
             value: /secret/gcp-credentials/key.json
           - name: CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE
             value: /secret/gcp-credentials/key.json
+          - name: WANDB_NAME
```

Comment:
It might also be useful to group trainings under something like WANDB_RUN_GROUP={RUNDATE}-{GROUP_NUMBER}.  I've found the sorting by job-type / group has been helpful so far, but prognostic runs are much more numerous, so it might not add much I guess.

Reply:
I found searching on the unique "group" suffix makes sense. Currently, we use groups to mean groups of the jobs involved in a single "trial" of an experiment (e.g. all the prog runs for a single model). Perhaps using tags is the way to go since we can change them after the fact, nest tags within other tags, etc.